scripts/extract_validation_messages.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+"""Extract validation messages from the frameworks questions content.
+
+This script retrieves all the validation messages within the frameworks repo for
+all the apps. There are small number of validation messages in the frontend apps
+which will need extracting manually for completeness.
+
+Specify the output path for the files as below, and a directory will be created
+with a separate yaml file for each framework, containing that frameworks
+validation messages.
+
+Usage:
+    extract_validation_messages.py --output-path=<output_path>
+
+"""
+import yaml, os, shutil, sys
+from docopt import docopt
+
+def check_and_create_output_directory(path):
+    if os.path.exists('{}/validation_messages'.format(path)):
+        print('{}/validation_messages already exists. Please specify a different output path.'.format(path))
+        sys.exit()
+
+    os.makedirs('{}/validation_messages'.format(path))
+
+def create_data_packet(file_path, doc):
+    return dict(
+        path=file_path.split('..')[-1],
```

Comment:
Can see why you're doing this but... `bit odd` to be specifically depending on the `../` being in the path in the right place. Just a bit fragile to future adaptations (no, I don't have a suggested alternative...)